.travis.yml

```diff
@@ -48,6 +48,21 @@ jobs:
       if: type != pull_request
       os: osx
 
+    - env:
+      - ENV_FILE=continuous_integration/travis/travis-mindeps-array-dataframe.yaml
+      - *test
+      - *no_coverage
+      - *no_optimize
+      - *no_imports
+
+    - env:
+      - ENV_FILE=continuous_integration/travis/travis-mindeps-bag-delayed.yaml
+      - *test
+      - *no_coverage
+      - *no_optimize
+      - *no_imports
```

Comment:
I see now that we're adding two more jobs to the CI system.  I don't think that verifying that we work without cloudpickle is a big enough win to justify the added CI time.  I would prefer if these were removed.

Reply:
Thoughts?

Reply:
It would prevent the repeating of pydata/xarray#3401, although test_imports.sh slightly mitigates the risk.

Note that these two new test suites are much faster than the other ones, in virtue that they're skipping a load of slow tests.

Even if I were to agree that it's OK not to have any tests for when cloudpickle isn't installed (and I don't), you'll need the extra test suite anyway for #5564.

Reply:
I think that neither #5564 nor https://github.com/pydata/xarray/issues/3401 are sufficient to justify an additional CI job.  These are, in my book, relatively minor problems (almost no one complains about them) while CI congestion has been a very real problem in the past (many people have complained about it).  

We could do something like what we do for HDFS, which is something like a nightly job I think (although I don't know the CI system well anymore).  HDFS was in the same position of needing to be tested, but not important enough to justify the regular CI run.  It's probably a good model to look at for this.

Reply:
Changed the two new test suites to behave like the one for HDFS